Traceback (most recent call last):
  File "/home/users/wx83/GNN_baseline/GNN/train.py", line 72, in <module>
    NUM_FEAT = next(iter(train_loader))['data'].x.shape[1]
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/users/wx83/miniconda3/envs/gnn/lib/python3.11/site-packages/torch_geometric/data/hetero_data.py", line 156, in __getattr__
    raise AttributeError(f"'{self.__class__.__name__}' has no "
AttributeError: 'HeteroDataBatch' has no attribute 'x'
